packages/melos/lib/src/commands/bootstrap.dart

```diff
@@ -109,6 +109,16 @@ mixin _BootstrapMixin on _CleanMixin {
     required bool noExample,
   }) async {
     final filteredPackages = workspace.filteredPackages.values;
+    final isCI = utils.isCI;
+
+    bool parallelism;
```

Comment:
The code would be a bit easier to read with a switch assignment here

Reply:
good call!